src/charts/vertical_bar.coffee

```diff
@@ -344,13 +354,15 @@ Ember.Charts.VerticalBarComponent = Ember.Charts.ChartComponent.extend(
     # the origin line so that they do not overlap with it
     zeroDisplacement = 1
     yScale = @get 'yScale'
-    class: (d,i) -> "grouping-#{i}"
+    class: (d,i) =>
```

Comment:
Same problem with the variable names.

Sometimes `d` is a label, sometimes it's an object. It's confusing